### DISCUSSION TITLE
Allow changing the icon size

### DISCUSSION THREAD

**Discussion by roberteggl** [callstack/react-native-bottom-tabs#215] [Ideas]
### Why it is needed?

I use the SF symbols, but they seem a bit too big in the bottom bar view. It would be nice if you could adjust the size.

### Possible implementation

Add a new prop to the tabbar or to each symbol

**Comment by okwasniewski** [comment]
Thanks for opening the feature request. I'm moving this to discussions because all feature requests will now be under the "Ideas 💡" tab in GitHub discussions.

**Comment by skdrew** [comment]
Hi, I was wondering if there's been any thought put into this feature? I've been playing around with the library and would really like to implement it in our app, but we're having trouble getting the icon sizes to match our existing implementation.

I can't find any documentation on what the ideal image size should be for the tabs for either iOS or Android, and we're getting inconsistent results. We're using an svg as the icon image, which is set to 20x20 in the xml. On iOS it looks sharp, though too big for our designs (so would really like to be able to specify the icon size). However on Android it's rendering as though it's a lo-dpi image. If we update the icon to 40x40 it looks sharp, which has thrown me a bit as I'd have expected it to scale the svg accordingly.

Many thanks, happy to help out with this one wherever possible!
